src/libpriv/rpmostree-postprocess.c

```diff
@@ -1328,11 +1328,49 @@ rpmostree_copy_additional_files (GFile         *rootfs,
   return ret;
 }
 
+static char *
+mutate_os_release (const char    *contents,
+                   const char    *base_version,
+                   const char    *next_version)
+{
+  char **lines = NULL;
+  GString *new_contents = g_string_sized_new (strlen (contents));
+
+  lines = g_strsplit (contents, "\n", -1);
+  for (char **it = lines; it && *it; it++)
+    {
+      if (strlen (*it) == 0)
```

Comment:
I normally do something like: `const char *line = *it;` for the iterator point if referencing more than once.